### PR TITLE
MBSA-708 Create fake emcee queue server

### PR DESCRIPTION
**Pull request by sergioboi**
The first step where I create and test locally simple Ktor server

### CODE REVIEW

File: subprojects/emcee/queue-fake-server/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Fake emcee server
```
Comment (sergioboi) [inline subprojects/emcee/queue-fake-server/README.md:1]: Describe all useful info about `Ktor` that I found

File: subprojects/emcee/queue-fake-server/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+```
+
+```shell
+./gradlew :subprojects:emcee:queue-fake-server:run
+```
+
+## How to test
+
+Ask me for sharing POSTMAN files
```
Comment (RuslanMingaliev) [inline subprojects/emcee/queue-fake-server/README.md:19]: Why can't we test it with the real worker instance?

File: subprojects/emcee/queue-fake-server/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## How to build Docker image
+
+[Ktor docker image sample](https://github.com/ktorio/ktor-samples/tree/main/docker-image)
+
+We use Gradle plugin `com.google.cloud.tools.jib`
```
Comment (RuslanMingaliev) [inline subprojects/emcee/queue-fake-server/README.md:25]: Why did you decide to use this plugin? Wouldn't it be better to create our own image as we did for `android-builder` and `android-emulator`? Then we'll replace only one line of code `java -jar fake-queue.jar` with real `./Emcee ...`.

File: subprojects/emcee/queue-fake-server/build.gradle.kts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+application {
+    mainClass.set("com.avito.emcee.server.fake.MainKt")
+}
+
+
+dependencies {
+    implementation(projects.subprojects.emcee.queueClientApi)
+    implementation("ch.qos.logback:logback-classic:1.2.11")
```
Comment (RuslanMingaliev) [inline subprojects/emcee/queue-fake-server/build.gradle.kts]: What is this for?

Comment (sergioboi) [reply subprojects/emcee/queue-fake-server/build.gradle.kts]: I will add because

**Review by RuslanMingaliev — APPROVED** [review]

**Pull request merged**